app/components/branch-row.js

```diff
@@ -26,6 +27,15 @@ export default Component.extend({
     return this.externalLinks.commitUrl(vcsType, { owner, repo, commit });
   }),
 
+  vcsType: computed('branch.repository.id', function () {
+    const repository = this.store.peekRecord('repo', this.get('branch.repository.id'));
+    return repository.vcsType;
+  }),
+
+  provider: computed('vcsType', function () {
+    return this.get('vcsType') && this.get('vcsType').toLowerCase().replace('repository', '');
```

Comment:
I think in the long run it would be nice to have a standard, centrally located way to do this. Maybe something in  `app/utils/vcs.js`?

I think in one case what I did was get the provider config with one of those utils, and then read the provider's `urlPrefix`. But maybe a util like `vcsUrlPrefix` would be useful, to make it just like a single function to call. Or possibly we shouldn't rely on the urlPrefix, but rather add a `slug` property or something for this exact purpose.

Anyway, these are some optional ideas. Not really necessary for launch :+1: